lib/matplotlib/backend_bases.py

```diff
@@ -1365,8 +1365,10 @@ def __init__(self, name, canvas, x, y, guiEvent=None):
         *x*, *y* in figure coords, 0,0 = bottom, left
         """
         Event.__init__(self, name, canvas, guiEvent=guiEvent)
-        self.x = x          # x position - pixels from left of canvas
-        self.y = y          # y position - pixels from right of canvas
+        # x position - pixels from left of canvas
+        self.x = int(x) if x is not None else 0
```

Comment:
We should probably leave the current behavior of x and y being allowed to be None.  I am not sure why that is allowed, but we should not change it!